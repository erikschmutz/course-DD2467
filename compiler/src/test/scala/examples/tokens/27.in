type 'a tree = Leaf | Node of 'a * 'a tree * 'a tree