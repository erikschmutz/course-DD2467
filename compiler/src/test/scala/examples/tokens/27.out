TYPE()TYPE_PARAM('a)IDENTIFIER(tree)EQUAL()CONSTR_IDENTIFIER(Leaf)LINE()CONSTR_IDENTIFIER(Node)OF()TYPE_PARAM('a)MULTIPLY()TYPE_PARAM('a)IDENTIFIER(tree)MULTIPLY()TYPE_PARAM('a)IDENTIFIER(tree)